src/compiler/scala/tools/nsc/transform/Mixin.scala

```diff
@@ -143,6 +143,7 @@ abstract class Mixin extends InfoTransform with ast.TreeDSL {
       // implementation class, as it's a clone that was made after erasure, and thus it does not
       // know its info at the beginning of erasure anymore.
       val sym = mixinMember cloneSymbol clazz
+      sym.removeAttachment[NeedStaticImpl.type]
```

Comment:
is it this? even though we're in preTransform, the mixin member (of a different compilation unit) might already have been transformed by mixin's postTransform and therefore have the attachment. we remove it from the forwarder.